### ISSUE TITLE
Execute button not working due to Schema Valication

### ISSUE THREAD

**Issue by vishmit** [swagger-api/swagger-ui#8085]
If a JSON request body does not validate against its schema (e.g. a required property is missing, or some property value does not match minLength or pattern), the "Execute" button does not work, it does nothing. 
Do we have an option to disable this validation while configuring the SwaggerUI client.

**Comment by suipaste** [comment]
Yes this or a validation message to explain what is failing validation.
